G/GMP/GMP@6.3.0/bundled/patches/gmp-exception.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/errno.c b/errno.c
+index b4be555..3f772a5 100644
+--- a/errno.c
++++ b/errno.c
+@@ -68,5 +68,8 @@ __gmp_sqrt_of_negative (void)
+ void
+ __gmp_divide_by_zero (void)
+ {
++  /* try to force a division by zero system exception */
++  __gmp_junk = 10 / __gmp_0;
```

Comment:
Note that by default, GMP now uses `__gmp_exception` which in turn does this:
```
/* Use SIGFPE on systems which have it. Otherwise, deliberate divide
   by zero, which triggers an exception on most systems. On those
   where it doesn't, for example power and powerpc, use abort instead. */
void
__gmp_exception (int error_bit)
{
  gmp_errno |= error_bit;
#ifdef SIGFPE
  raise (SIGFPE);
#else
  __gmp_junk = 10 / __gmp_0;
#endif
  abort ();
}
```
I wonder: instead of patching GMP to force division by zero here, could/should we catch SIGFPE instead?

Note that this also affects the overflow func patch, see my other comment